source/AssemblyInfo.cs

```diff
@@ -7,3 +7,4 @@
 [assembly: AssemblyMetadata ("BUILD_TIMESTAMP", "{BUILD_TIMESTAMP}")]
 
 [assembly: Android.LinkerSafe]
```

Comment:
`[assembly: LinkerSafe]` will be a warning in .NET 6:

https://github.com/xamarin/xamarin-android/blob/e4debf72015549e6e19cbfb247a47fead6ce7a01/src/Mono.Android/Android/LinkerSafeAttribute.cs#L5-L7

You could `#if` around it for .NET 6. But there are probably so many warnings, probably not seeing it anyway.

Reply:
👍 I think that warning isn't showing up yet since `[LinkerSafe]` won't be `[Obsolete]` until P7.

Wrapped it in `#if`.